### PR TITLE
[2.7] bpo-36417: Fix bug in unicode.isdecimal documentation, add isdigit documentation

### PR DESCRIPTION
**Pull request by zheng**
https://bugs.python.org/issue36417

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Our records indicate we have not received your CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Comment by tirkarthi** [comment]
Isn't `Lib/pydoc_data/topics.py` generated by release manager as part of a release?

**Comment by zheng** [comment]
That makes sense! I got rid of the changes

File: Doc/library/stdtypes.rst [hunk 1]
```diff
@@ -1406,12 +1406,21 @@ The following methods are present only on unicode objects:
    that have the Unicode numeric value property, e.g. U+2155,
    VULGAR FRACTION ONE FIFTH.
 
+.. method:: unicode.isdigit()
+
+   Return ``True`` if there are only digit characters in S, ``False``
```
Comment (pganssle) [inline Doc/library/stdtypes.rst:1411]: This method doesn't refer to `S` anywhere else, presumably the "in S" thing can be replaced with "in the string" or "in the unicode string"? 

I imagine you can just copy the exact wording from [the 3.7 documentation](https://docs.python.org/3.7/library/stdtypes.html#str.isdigit) (seems like that's mostly what this is, but with the "in S" thing for consistency with the other methods).

File: Doc/library/stdtypes.rst [hunk 2]
```diff
@@ -1406,12 +1406,21 @@ The following methods are present only on unicode objects:
    that have the Unicode numeric value property, e.g. U+2155,
    VULGAR FRACTION ONE FIFTH.
 
+.. method:: unicode.isdigit()
```
Comment (vadmium) [inline Doc/library/stdtypes.rst:1409]: Wouldn't it make more sense to put this next to the _str.isdigit_ method, like is done with the _splitlines_ methods? It doesn't really belong in this section.

**Comment by zware** [comment]
Hi @zheng!  Unfortunately, this did not reach a conclusion before Python 2.7 reached EOL on January 1st.  As this does not appear to be a critical security issue, there is almost no chance that it will be accepted in this brief limbo window between end-of-support and the final 2.7.18 release, so I'm going to go ahead and close it.

Thanks for your contribution anyway, and I hope your next one is more fruitful!

**Pull request closed without merging**